CacheTagsBundle/Annotation/CacheTag/AbstractCacheTag.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+namespace lbarulski\CacheTagsBundle\Annotation\CacheTag;
+
+use Doctrine\Common\Annotations\Annotation\Required;
+use Doctrine\Common\Annotations\Annotation\Target;
+use lbarulski\CacheTagsBundle\Tag\TagInterface;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\ConfigurationInterface;
+
+/**
+ * @Annotation
+ * @Target("METHOD")
+ */
+abstract class AbstractCacheTag implements TagInterface, ConfigurationInterface
+{
+	const ALIAS_NAME = 'cache_tag';
```

Comment:
this should be defined on `Plain` and `RequestAttribute` classes explicitly and `getAlias` shouldn't be defined here I think